CHANGELOG.rst

```diff
@@ -1,6 +1,16 @@
 Changelog
 =========
 
+1.2.0 (2021-09-06)
+------------------
+
+- [NEW] Added Albanian, Tamil and Zulu locales.
+- [NEW] Added support for ``Decimal`` as input to ``arrow.get()``.
+- [FIX] The Arabic, Czech, Finnish, Hebrew, Icelandic, Slavic, Slovak and Thai locales now support ``dehumanize``.
```

Comment:
Finnish, Nepali, Estonian and Zulu are the only new locales that support dehumanize at the moment

Reply:
Ahh ok so the others have been refactored in preparation for supporting it?

Reply:
Correct.